apps/aecore/src/aec_connection_sup.erl

```diff
@@ -85,3 +101,13 @@ sync_listen_address() ->
                 aecore, sync_listen_address, ?DEFAULT_SYNC_LISTEN_ADDRESS),
     {ok, IpAddress} = inet:parse_address(binary_to_list(Config)),
     IpAddress.
+
+acceptors() ->
+    aeu_env:user_config_or_env([<<"sync">>, <<"accetors">>],
```

Comment:
Does accetors pick correct value here?

Reply:
I guess it should be `acceptors` (typo) ?

Reply:
It seems typo.